schemas/account.js

```diff
@@ -0,0 +1,21 @@
+const email = {
+  type: 'string'
+};
+
+const postAccounts = {
+  title: 'bedrock-accounts-http account creation post',
```

Comment:
Add: `additionalProperites: false` where appropriate, otherwise, the default is that other properties are allowed.

https://json-schema.org/understanding-json-schema/reference/object.html